features/step_definitions/file_steps.coffee

```diff
@@ -4,18 +4,22 @@ path = require 'path'
 
 module.exports = ->
 
-  @Given /^I have no test files$/, (done) ->
-    done() # No op for readability
+  @Given /^my project has no test files$/, (done) ->
+    done()
 
 
-  @Given /^I have 1 (passing|failing) test$/, (status, done) ->
-    outputTestFile path.join(@tmpDir, 'example_spec.coffee'), passing: status is 'passing', done
+  @Given /^my project has a (passing|failing) test$/, (status, done) ->
+    outputTestFile path.join(@tmpDir, 'example_spec.coffee'),
+                   passing: status is 'passing',
+                   done
```

Comment:
The reason why I indented this in the first place is that this is confusing to read when on one line, and I'd like to separate the arguments visually here.

Coming to a common understanding of allowed styles is necessary at some point. So I'd rather figure this out now than ignore it, because then we (or somebody else) runs into this exact same issue again in the future. That's an important growth step for us as a team and this project.

I gave you a number of good arguments for choosing my variation of formatting in this particular case, including a compromise to use your style when that leads to better readability. Allowing several ways of formatting complex expressions isn't uncommon, since that is often dependent on the surrounding code flow. If none of that convinces you, then fine. You are the maintainer of this repo right now, so its your call.